### PR TITLE
update to the latest version of ansible-runner

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,48 +1,48 @@
 adal==1.2.2               # via msrestazure
-aiohttp==3.6.2
+aiohttp==3.6.2            # via -r /awx_devel/requirements/requirements.in
```
Comment (ryanpetrello) [inline requirements/requirements.txt:2]: Almost none of this (other than the runner version) represents an actual change; it's just a change in the way the tooling generates comments now cc @AlanCoding.

**Comment by ryanpetrello** [comment]
running through some integration before merging this

File: requirements/requirements.txt (hunk 1 above)
Comment (AlanCoding) [reply requirements/requirements.txt:2]: this still doesn't apply the change to requirements_ansible.txt

I'm fully in favor of applying this, it's an intended improvement.

**Comment by AlanCoding** [comment]
https://github.com/ansible/ansible-runner/blob/devel/CHANGES.rst

```
1.4.5 (2020-03-19)
Fix an issue with --process_isoloation_*_ paths parsing cli args
Switch default docker images to centos:8
Switch extravar format so we can support more than just string types
Make sure job events folder is created earlier to prevent errors when using immediately after starting a runner job
Annotate all runner_on events with start/end/duration times
```

File: requirements/requirements_ansible.txt [hunk 2]
```diff
@@ -1,46 +1,46 @@
 adal==1.2.2               # via msrestazure
-apache-libcloud==2.5.0
+apache-libcloud==2.5.0    # via -r /awx_devel/requirements/requirements_ansible.in
```
Comment (ryanpetrello) [inline requirements/requirements_ansible.txt:2]: cc @AlanCoding

**Review by AlanCoding — APPROVED** [review]

**Comment by ryanpetrello** [comment]
@wenottingham do you have some AWX issue I don't know about to track this work?

https://github.com/ansible/ansible-runner/pull/405

**Review by Spredzy — APPROVED** [review]

**Comment by jakemcdermott** [comment]
recheck

**Comment by ryanpetrello** [comment]
regate

**Comment by wenottingham** [comment]
> @wenottingham do you have some AWX issue I don't know about to track this work?
> 
> [ansible/ansible-runner#405](https://github.com/ansible/ansible-runner/pull/405)

It's for analytics usage (tracking task timings, which only existed for a subset of modules beforehand.0

**Pull request merged**